pom.xml

```diff
@@ -25,7 +25,7 @@
         <connection>scm:git:ssh://git@github.com/TechnologyBrewery/orphedomos.git</connection>
         <developerConnection>scm:git:ssh://git@github.com/TechnologyBrewery/orphedomos.git</developerConnection>
         <url>scm:git:ssh://git@github.com/TechnologyBrewery/orphedomos</url>
-      <tag>HEAD</tag>
+      <tag>orphedomos-parent-0.10.0</tag>
```

Comment:
@d-ryan-ashcraft is this correct for orphedomos-parent?

Reply:
This probably should be back to head, but I don't think it really matters